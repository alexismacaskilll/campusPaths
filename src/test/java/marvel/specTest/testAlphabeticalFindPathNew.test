# Test the BFS picks alphabetically least path on a file that lists the heros and books
# in reverse alphabetical order

# Load a graph from tsv file that contains heros and comics in reverse alphabetical order.
LoadGraph graph1 alphabeticalNew.tsv

# Find shortest path from hero 1 to hero 4
FindPath graph1 hero_1 hero_4