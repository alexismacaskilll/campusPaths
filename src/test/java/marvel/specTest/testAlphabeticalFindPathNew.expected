# Test the BFS picks alphabetically least path on a file that lists the heros and books
# in reverse alphabetical order

# Load a graph from tsv file that contains heros and comics in reverse alphabetical order.
loaded graph graph1

# Find shortest path from hero 1 to hero 4
path from hero 1 to hero 4:
hero 1 to hero 2 via book 1
hero 2 to hero 4 via book 3